tests/QS/regtest-dcdr/TEST_FILES.toml

```diff
@@ -12,4 +12,5 @@
 "h2o_apt_pbc.inp"                       = [{matcher="M096", tol=1e-06, ref=-0.938707}]
 "h2o_apt_pbc_loc.inp"                   = [{matcher="M096", tol=1e-06, ref=-0.938770}]
 "h2o_aat.inp"                           = [{matcher="M100", tol=1e-06, ref=-0.857427}]
+"h2o_apt_fdiff.inp"                     = [{matcher="M127", tol=1e-04, ref=0.0034319}]
```

Comment:
The matcher "M127" does not exist. Did you maybe forget to include your changes of [matchers.py](https://github.com/cp2k/cp2k/blob/master/tests/matchers.py) in this PR?